test/functionalTest/cases/1949_permanent_subscriptions/1949_permanent_subscriptions.test

```diff
@@ -0,0 +1,240 @@
+# Copyright 2016 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
I would suggest to add a test to check that update/delete expiration works, I mean:

1. Create regular subscription (with expires)
2. Get subscription to see the 'expires' field
3. Update subscription to make it permanent
4. Get subscription to check that 'exipres' field is not shown
5. Update subscription to provide a expiration date again
6. Get subscription to see the 'expires' field